examples/middleware_factory/sam/combining_powertools_utilities_template.yaml

```diff
@@ -0,0 +1,136 @@
+      - Version: '2012-10-17' # Policy Document
+        Statement:
+          - Effect: Allow
+            Action:
+              - dynamodb:PutItem
+            Resource: !GetAtt HistoryTable.Arn
+          - Effect: Allow
+            Action: # https://docs.aws.amazon.com/appconfig/latest/userguide/getting-started-with-appconfig-permissions.html
+              - ssm:GetDocument
+              - ssm:ListDocuments
+              - appconfig:GetLatestConfiguration
+              - appconfig:StartConfigurationSession
+              - appconfig:ListApplications
+              - appconfig:GetApplication
+              - appconfig:ListEnvironments
+              - appconfig:GetEnvironment
+              - appconfig:ListConfigurationProfiles
+              - appconfig:GetConfigurationProfile
+              - appconfig:ListDeploymentStrategies
+              - appconfig:GetDeploymentStrategy
+              - appconfig:GetConfiguration
+              - appconfig:ListDeployments
+              - appconfig:GetDeployment
+            Resource: "*"
+      Events:
+        GetComments:
+          Type: Api
+          Properties:
+            Path: /comments
+            Method: GET
+        GetCommentsById:
+          Type: Api
+          Properties:
+            Path: /comments/{comment_id}
+            Method: GET
+
+  # DYNAMODB TABLE FOR HISTORIC
+  HistoryTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: "HistoryTable"
+      AttributeDefinitions:
+        - AttributeName: customer_id
+          AttributeType: S
+        - AttributeName: request_id
+          AttributeType: S
+      KeySchema:
+        - AttributeName: customer_id
+          KeyType: HASH
+        - AttributeName: request_id
+          KeyType: "RANGE"
+      BillingMode: PAY_PER_REQUEST
+
+  # APPCONFIG FOR FEATURE FLAGS
+  FeatureCommentApp:
+    Type: AWS::AppConfig::Application
+    Properties:
+      Description: "Comments Application for feature toggles"
+      Name: comments
+
+  FeatureCommentDevEnv:
+    Type: AWS::AppConfig::Environment
+    Properties:
+      ApplicationId: !Ref FeatureCommentApp
+      Description: "Development Environment for the App Config Comments"
+      Name: dev
+
+  FeatureCommentConfigProfile:
+    Type: AWS::AppConfig::ConfigurationProfile
+    Properties:
+      ApplicationId: !Ref FeatureCommentApp
+      Name: features
+      LocationUri: "hosted"
+
+  HostedConfigVersion:
+    Type: AWS::AppConfig::HostedConfigurationVersion
+    Properties:
+      ApplicationId: !Ref FeatureCommentApp
+      ConfigurationProfileId: !Ref FeatureCommentConfigProfile
+      Description: 'A sample hosted configuration version'
+      Content: |
+        {
+              "save_history": {
+                "default": true
+              }
+        }
+      ContentType: 'application/json'
+
+  # this is just an example
+  # change this values according your deployment strategy
+  BasicDeploymentStrategy:
```

Comment:
I added the deployment strategy to avoid the default baking time.. Too long if someone wants to test this template.